packages/web-components/fast-foundation/docs/guide/blazor.doc.md

```diff
@@ -3,4 +3,102 @@ id: blazor
 title: Blazor
 sidebar_label: Blazor
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-foundation/docs/guide/blazor.doc.md
----
\ No newline at end of file
+---
+
+FAST-DNA works seamlessly with ASP.NET Blazor, including integration with Blazor's binding engine and components. Let's take a look at how to set things up.
+
+## Setting up the Blazor Project
+
+First, you'll need to make sure that you have the .NET SDK installed. You can learn more and download that [on the official site](https://dotnet.microsoft.com/download).
+
+With the SDK installed, you have access to the `dotnet` command line interface. This can be used to create a new ASP.NET Blazor project. For example, to create a new ASP.NET Blazor App named "fast-blazor", you would use the following command:
```

Comment:
```suggestion
With the SDK installed, you have access to the `dotnet` command line interface. This can be used to create a new Blazor project. For example, to create a new Blazor App named "fast-blazor", you would use the following command:
```